pages/dashboard.vue

```diff
@@ -78,7 +78,9 @@ export default {
   },
   computed: {
     authUrl() {
-      return `https://api.modrinth.com/api/v1/auth/init?url=https://modrinth.com${this.$route.fullPath}`
+      return `https://api.modrinth.com/api/v1/auth/init?url=${
+        process.env.domain || window.location.origin
```

Comment:
Are you sure this is right logic here? Perhaps null coalescing operator is better (?? operator). || operator will default to the right hand side on any falsey value, including `''` and `0`, while ?? operator will only do so on `null` or `undefined` value

Reply:
It doesn't really matter, because `process.env.domain` would never be `''` or `0`. Though, maybe it isn't even needed. [This](https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-env/) makes it sound like environment variables are available client side.

Reply:
Turns out `window.location.origin` isn't needed. I removed it in b44fdc0.